.github/workflows/android_apk_build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build Android APK debug file
+
+on:
+  push:
+    paths:
+    - 'android/**'
+    - '.github/workflows/android_apk_build.yaml'
+
+jobs:
+  build:
+    name: Checkout and package
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout to push branch
+        uses: actions/checkout@master
+        with:
+          ref: ${{ github.ref }}                                                              # checkout to branch which was pushed
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build APK file
+        if: job.status == 'success'
+        run: |
+          cd android
+          wget https://eaterslab.herokuapp.com/api.yaml
+          chmod +x ./api_generate_kotlin.sh
+          ./api_generate_kotlin.sh api.yaml
+          export GRADLE_USER_HOME=$(pwd)/.gradle
+          chmod +x ./gradlew
+          ./gradlew assembleDebug
+      - name: Upload
```

Comment:
Could you add a comment how it works? Does it commit to master/pushed branch, or upload to releases?

Reply:
for now, all my workflows are lauched on any branch. Will add comments